src/Repository/CommitteeMembershipRepository.php

```diff
@@ -417,4 +417,24 @@ public function findCommitteesUuid(array $criteria): array
             return $uuid->toString();
         }, array_column($qb->getQuery()->getArrayResult(), 'uuid'));
     }
+
+    public function findCommitteesByPrivilege(Adherent $adherent, array $privilege): array
+    {
+        // Prevent SQL query if the adherent doesn't follow any committees yet.
+        if (0 === \count($adherent->getMemberships())) {
+            return [];
+        }
+
+        $qb = $this->createQueryBuilder('cm');
+
+        return $qb
```

Comment:
You can remove the `$qb` variable and return directly : 

```php 
return $this
    ->createQueryBuilder('cm')
    ->addSelect('c.id, c.name, c.slug')
    // ...
```

Reply:
i need to use ```$qb``` for the ```->where($qb->expr()->in('cm.privilege', $privilege))```

Reply:
Yes okay :wink: